core/src/main/scripts/importer/validateData.py

```diff
@@ -1275,14 +1274,24 @@ def checkMatchedNormSampleBarcode(self, value):
     def checkVerificationStatus(self, value):
         # if value is not blank, then it should be one of these:
         if self.checkNotBlank(value) and value.lower() not in ('verified', 'unknown', 'na'):
-            return False
+            self.logger.warning(
+                "Value in 'Verification_Status' not in MAF format",
+                extra={'line_number': self.line_number,
+                       'cause':value})
+            # return without error (just warning above)
+            return True
         return True
 
     def checkValidationStatus(self, value):
         # if value is not blank, then it should be one of these:
         if self.checkNotBlank(value) and value.lower() not in ('untested', 'inconclusive',
                                  'valid', 'invalid', 'na', 'redacted', 'unknown'):
-            return False
+            self.logger.warning(
+                "Value in 'Validation_Status' not in MAF format",
```

Comment:
checked some parts of the code like `ValidationStatusColumnFormatter.tsx` and it seems to handle unofficial values well, so OK. Backend code seems to ingest anything (except `wildtype` in `ImportExtendedMutationData.java`...which we could also warn about here), so that is also OK.